src/main/java/cinema/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package cinema.controllers;
+
+import cinema.model.dto.UserRequestDto;
+import cinema.service.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/register")
```

Comment:
let's move it to method, probably you'll have @RequestMapping("/login") method in future in this particular controller